source/_integrations/ecoforest.markdown

```diff
@@ -59,3 +59,10 @@ The Ecoforest integration exposes multiple sensors to monitor various features:
 - ntc_temperature: a sensor for the current <abbr title="negative temperature coefficient">NTC</abbr> probe temperature
 - status: a sensor for the current status of the device. Possible values are: off, starting, pre-heating, on, shutting down, standby, alarm.
 - alarm: a sensor for the current alarm of the device. Possible values are: air depression, pellets, CPU overheating, unknown.
+- depression: a sensor for the current depression air entrance
+- working_hours: a sensor with the duration working time of the device
```

Comment:
```suggestion
- working_hours: a sensor with the duration working time of the device
```
what is meant here? The total number of working hours since the device was first used?

Reply:
yes